app/externalapi.py

```diff
@@ -0,0 +1,50 @@
+
+import requests
+
+def headlines_feed():
+
+    query_params = {
+        "sources": "bbc-news,the-verge",
+        "apiKey": "e90358389f0b41b2bf25bfc55c92f69c"
+    }
+
+    main_url = "https://newsapi.org/v2/top-headlines"
+    my_headlines = requests.get(main_url, params=query_params)
+    open_page = my_headlines.json()
+    print(open_page)
+    return open_page
+
+#@app.route('/allnews')
+def all_news_feed():
+
+    query_params = {
+        'q': 'big data',  # query phrase
```

Comment:
So all articles you look for are articles that contain "big data"? Seems like a peculiar hardcoding here.